docs/adguard-for-windows/overview.md

```diff
@@ -49,33 +49,36 @@ Many websites gather information about their visitors, such as their IP addresse
  
 You can flexibly adjust the work of Stealth Mode: for instance, you can prohibit the website to receive the search request you used to find it on the Internet, automatically delete both third-party and website’s own cookies, and disable location sharing in your browser that can be used to track your whereabouts.
  
-To learn all the main features and options you can manage in the Stealth mode settings, [read this article](/general/stealth-mode.md).  
+To learn everything about Stealth Mode and its many options, [read this article](/general/stealth-mode.md).  
 
 ### Browsing Security
 
-Phishing is a very common web fraud aimed at stealing your personal data (passwords, bank account details, etc.). It is carried out using fake web pages mimicking popular websites or real services, such as web stores or social networks. Many phishing websites have URLs that only differ from the real ones in just a few symbols to put off your guard. If a user enters their details on such a website, they will definitely end up in the hands of criminals.
-
-Browsing Security gives strong protection against such incidents. In addition, the regularly updated module bases allow it to recognize malicious websites, thus securing your PC from viruses and trojans. At the same time, the user’s personal information is not transferred anywhere, and AdGuard servers do not know which websites the user is visiting, because security check is not performed via open web address (URL) but through hash prefixes (hash is a certain data structure that renders each address added to the base unambiguous). Some web pages are not malicious themselves, but they may contain elements built into their structure by other websites. Unlike browser plug-ins, Browsing Security module checks each object built into any web page, thus ensuring maximum protection.
-
->But note that AdGuard for Android is not an antivirus. It will neither stop the actual download of a virus nor delete the already existing ones. For the complete protection of your device, we recommend using AdGuard in tandem with a dedicated AV tool.
+Browsing Security gives strong protection against malicious and fishing websites. No, AdGuard for Windows is not an antivirus. It will neither stop the actual download of a virus nor delete the already existing ones. But it will prevent downloading the content of a website whose domain has been added to our untrusted sites database. How does this module work, you can read in the [dedicated article](/general/stealth-mode.md).
```

Comment:
Странно звучит - не остановит загрузку, но остановит загрузку. Давай так:

>Browsing Security gives strong protection against malicious and fishing websites. No, AdGuard for Windows is not an antivirus. It will neither stop the download of a virus when it's already started, nor delete the already existing ones. But it will warn you if you're about to proceed to a website whose domain has been added to our "untrusted sites" database, or to download a file from such website. You can find more information about how this module works in the [dedicated article](/general/stealth-mode.md).